pkg/envtest/server.go

```diff
@@ -136,7 +136,7 @@ type Environment struct {
 	// CRDs is a list of CRDs to install.
 	// If both this field and CRDs field in CRDInstallOptions are specified, the
 	// values are merged.
-	CRDs []client.Object
+	CRDs []apiextensionsv1.CustomResourceDefinition
```

Comment:
Just a doubt. `apiextensionsv1.CustomResourceDefinition` does seem apt, but was there any specific reason why `client.Object` was used previously ? (was it just because `v1beta1` crds were also supported)

Reply:
> was it just because v1beta1 crds were also supported

That's exactly it.